index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="portfolio.css">
+    <title>Nsanzimfura portfolio</title>
+
+</head>
+<body>
+    <header>
+        <nav class="top-navigation-bar">
+            <h2 class="header-welcome">Welcome</h2>
+        </nav>
+        <img class="menu-icon" src="./assets/header-files/icon-left-side-menu.svg" alt="menu icon" />
+    </header>
+    <main>
+        <section class="main-page">
+            <h3 class="introduction-text"><span class="dark-blue">Hey there. I'm <br>Eden<br></span>
+            <span class="green-content">I'm a software developer</span></h3>
+            <p class="description-profession">
+                I can help you build a product , feature or website Look through some
+                of my work and experience! If you like what you see and have a project
+                you need coded, don’t hesitate to contact me.</p>
+
+            <aside class="social-media">
+                <img src="./assets/social-media-icons/twitter.svg" alt="twitter icon"/>
+                <img src="./assets/social-media-icons/Linkedin.svg" alt="linkedin icon"/>
+                <img src="./assets/social-media-icons/medium.svg" alt="medium icon"/>
+                <img src="./assets/social-media-icons/github.svg" alt="github icon"/>
+                <img src="./assets/social-media-icons/angellist.svg" alt="angellist icon"/>
+            </aside>
```

Comment:
- [ ] As stated by the previous reviewer, since this is a list of elements, in this case, social media icons, it would be better to use the `<ul>` tag (which represents an unordered list) than an `<aside>` tag. 🗒️  This not only improves the semantics of your project but makes it more accessible! 🤓  Also, since your intention with this list of icons is to provide the links to your social media, please wrap all of them with an anchor `<a>`! Kindly work on this 🛠️